jujugui/static/gui/src/app/components/vanilla/card/_card.scss

```diff
@@ -0,0 +1,59 @@
+%p-card {
```

Comment:
I have no idea what's going on in this file...

Reply:
This file is direct from the vanilla repo. I'm attempting to use vanilla components and add them to jaas as and when we build something new* that has been designed with vanilla components. Yes the VanillaCard component is used 3 times in one file but if we're to build features to designs accurately we need these reusable components - Spencer designs everything he can to vanilla specs and standards so it'll make things easier in the long run.

* rather than port everything over to a newer version of vanilla in one go.